runtime/commonMain/src/kotlinx/serialization/Encoding.kt

```
@@ -140,6 +141,13 @@ public interface Encoder {
      */
     public fun encodeByte(value: Byte)
 
+    /**
+     * Encodes an array of bytes as a CBOR ByteString.
+     *
+     * Can be ignored by all other formats.
+     */
+    public fun encodeByteString(value: ByteString) {}
```

Rather than an empty function, can we throw `UnsupportedOperation` (or similar) except for CBOR encoding?

---

Yeah, good idea. I'll update.